src/components/home/partners/Partners.jsx

```diff
@@ -1,17 +1,29 @@
-import PrimaryPartner from "./PrimaryPartner";
-import SecondaryPartner from "./SecondaryPartner";
-import CallToAction from "./CallToAction";
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import PrimaryPartner from './PrimaryPartner';
+import SecondaryPartner from './SecondaryPartner';
+import CallToAction from './CallToAction';
 
 const Partners = () => {
-  // any logic needed
   return (
     <>
+      <Box>
+        <Typography
+          component={'h2'}
+          variant="h3"
+          padding={{ xs: '50px 0 100px 0', md: '50px 50px 150px 50px' }}
+          display={'flex'}
+          textAlign={'center'}
+        >
+          Building a Better Tomorrow. With a shared vision, our mission would not be possible
```

Comment:
I know it's on the Figma this way, but I think these sentences are funky.
Maybe:
"Building a Better Tomorrow With a Shared Vision."
"Our mission would not be possible without our partners."

Reply:
changed in next push